<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRendererRulePropsWidget</class>
 <widget class="QWidget" name="QgsRendererRulePropsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editLabel"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="editDescription"/>
     </item>
     <item row="3" column="1">
      <widget class="QFrame" name="filterFrame">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editFilter"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnExpressionBuilder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="icon">
           <iconset resource="../../images/images.qrc">
            <normaloff>:/images/themes/default/mIconExpression.svg</normaloff>:/images/themes/default/mIconExpression.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTestFilter">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="mElseRadio">
       <property name="text">
        <string>Else</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="mFilterRadio">
       <property name="text">
        <string>Filter</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Catch-all for other features</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupScale">
     <property name="title">
      <string>Scale range</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QgsScaleRangeWidget" name="mScaleRangeWidget" native="true">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSymbol">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Symbol</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsScaleRangeWidget</class>
   <extends>QWidget</extends>
   <header>qgsscalerangewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editLabel</tabstop>
  <tabstop>mFilterRadio</tabstop>
  <tabstop>editFilter</tabstop>
  <tabstop>btnExpressionBuilder</tabstop>
  <tabstop>btnTestFilter</tabstop>
  <tabstop>mElseRadio</tabstop>
  <tabstop>editDescription</tabstop>
  <tabstop>groupScale</tabstop>
  <tabstop>groupSymbol</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mFilterRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterFrame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
